packages/integrations/vercel/src/serverless/adapter.ts

```diff
@@ -122,6 +128,34 @@ export interface VercelServerlessConfig {
 
 	/** The maximum duration (in seconds) that Serverless Functions can run before timing out. See the [Vercel documentation](https://vercel.com/docs/functions/serverless-functions/runtimes#maxduration) for the default and maximum limit for your account plan. */
 	maxDuration?: number;
+
+	/** Whether to cache on-demand rendered pages in the same way as static files. */
+	isr?: boolean | VercelISRConfig;
+}
+
+interface VercelISRConfig {
```

Comment:
I don't see the point of having a `VercelISRConfig` config where all the fields are optional. This means that a user can do:

```js
{
	isr: {}
}
```

Which feels wrong

Reply:
That is a possibility, but each one of the options really is optional and can individually be used. `isr: {}` would effectively be `isr: true`.

Reply:
JavaScript, what a language... 😔